src/templates/admin/cms/nav.html

```diff
@@ -21,21 +21,27 @@
             {% include "admin/elements/translations/form_tabs.html" with object=nav_item_to_edit %}
             <form method="POST">
                 {% csrf_token %}
-                <small>
-                    <p>To create a navigation link to a CMS page:</p>
-                    <ul>
-                        <li>{% if settings.URL_CONFIG == 'path' and request.journal %}Prefix the link with {{ request.journal.code }}/{% endif %}site/</li>
-                        <li>Add the URL Name of the page eg. about</li>
-                        <li>An example: {% if settings.URL_CONFIG == 'path' and request.journal %}Prefix the link with {{ request.journal.code }}/{% endif %}site/about/</li>
-                        <li>Link Name is the text that will appear in the nav bar. eg. About or Research Integrity</li>
-                        <li>For top level nav items that you don't want to appear in the dropdown leave the Link field empty.</li>
-                    </ul>
-                </small>
-                {% include "admin/elements/forms/errors.html" %}
-                {{ form|foundation }}
-                <button type="submit" name="submit" class="success button">
-                    {% if nav_item_to_edit  %}Update{% else %}Create{% endif %}
-                </button>
+                <div class="title-area">
+                    <h2>{% if nav_item_to_edit %}Currently Editing {{ nav_item_to_edit.link_name }}{% else %}Create New Nav Item{% endif %}</h2>
```

Comment:
rather than nav_item_to_edit, shall we check `form.instance`?
I think it is a bit more explicit and, in fact, I think this would be wrong in the case when a nav item is deleted? (there will be a `nav_item_to_edit` but `form.instance` will be `None`